packages/image/src/Image.story.js

```diff
@@ -3,110 +3,85 @@ import { storiesOf } from "@storybook/react";
 
 import thumbnailSizes from "./constants/thumbnailSizes";
 import badgeSizes from "./constants/badgeSizes";
-import imageTypes from "./constants/types";
 import Image from ".";
 import { withInfo } from "@storybook/addon-info";
 
 const imgSrc =
   "https://sourcewhatsgood.com/assets/images/utility_images/gallery-third-about-a518a29f64.jpg";
 
 storiesOf("Image", module)
-  .add("default", () => <Image src={imgSrc} />)
-  .add("thumbnail", () => <Image src={imgSrc} type={imageTypes.THUMBNAIL} />)
-  .add("badge", () => <Image src={imgSrc} type={imageTypes.BADGE} />)
+  .add("default", withInfo()(() => <Image src={imgSrc} />))
+  .add("thumbnail", withInfo()(() => <Image src={imgSrc} />))
+  .add("badge", withInfo()(() => <Image src={imgSrc} badge={true} />))
```

Comment:
When a property is a boolean, you don't need to explicitly set it to true.

You can write it like this: `<Image src={imgSrc} badge />`. If you don't pass the prop, it will be false.

Please refactor all occurrences of `badge={true}`;

Reply:
It's done.